src/main/java/sparksoniq/spark/udf/GroupClauseCreateColumnsUDF.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.spark.sql.RowFactory;
 import org.apache.spark.sql.api.java.UDF1;
 import org.joda.time.Instant;
+import org.omg.DynamicAny.DynAny;
```

Comment:
This looks interesting, what does this do?

Reply:
Nothing actually :-) It was merely a typo that somehow survived the import cleanups. Thanks for pointing it out !